src/TestInfrastructure/Framework/ConfigurableSolutionBindingInformationProvider.cs

```diff
@@ -1,4 +1,5 @@
-﻿//-----------------------------------------------------------------------
+﻿
```

Comment:
There should be no changes to this file.
Might be better to just revert the file, or remove the extra starting new-line and use VSCode to save the file as UTF-8 (_w/o_ BOM).
![image](https://cloud.githubusercontent.com/assets/5658207/14845776/6073df8c-0c58-11e6-82d6-6f45bc1efcd8.png)